src/cr/cube/cube.py

```diff
@@ -391,7 +391,7 @@ def valid_counts_summary(self):
         """ndarray of summary valid counts"""
         # --- In case of ndim > 2 the sum should be done on the second axes to get the
```

Comment:
This should go under the `if` condition, because it doesn't apply to it, but to the first line _in_ it.